driver/src/price_finding/naive_solver.rs

```diff
@@ -234,7 +247,9 @@ pub mod tests {
         let solver = NaiveSolver::new(None);
         let res = solver.find_prices(&orders, &state).unwrap();
 
-        assert_eq!(Some(u128_to_u256(16 * 10u128.pow(36))), res.objective_value);
+        // -28_000_000_000_000_100_000_000_000_000_000_000_000
```

Comment:
That's a shame. @bh2smith can we come up with an example or a small tweak to the simple solver logic to avoid not getting a good solution for this very simple example?

Reply:
Sorry, I am not sure I understand the request? What happened here?

Reply:
Oh, now that I see the example yields negative objective and is no good! I will see what we can do here.

Reply:
I've moved this to a separate issue #285, for this PR we can just ignore the objective value calculation in the naive solver.